docs/modules/ROOT/pages/erc4626.adoc

```diff
@@ -137,15 +137,22 @@ Following the previous math definitions, we have:
 | stem:[1+a_0+a_1]
 | stem:[10^\delta \times (1+a_0)]
 | stem:[10^\delta \times \frac{1+a_0}{1+a_0+a_1}]
+
+| after user's deposit
+| stem:[1+a_0+a_1+u]
+| stem:[10^\delta \times (1+a_0+a_1)]
+| stem:[10^\delta \times \frac{1+a_0+a_1}{1+a_0+a_1+u}]
 |===
 
 One important thing to note is that the attacker only owns a fraction stem:[\frac{a_0}{1 + a_0}] of the shares, so when doing the donation, he will only be able to recover that fraction stem:[\frac{a_1 \times a_0}{1 + a_0}] of the donation. The remaining stem:[\frac{a_1}{1+a_0}] are captured by the vault.
 
 [stem]
 ++++
-\mathit{loss} = \frac{a_1}{1+a_0}
+\text{loss} \approx \frac{a_1}{1 + a_0} - \frac{a_0 \times u}{1 + a_0}
```

Comment:
```suggestion
\text{loss} \approx \frac{a_1}{1 + a_0} - \frac{a_0 \times u}{1 + a_0} = \frac{a1 - a_0 \times u}{1 + a_0}
```

Reply:
loss is written using `mathit{loss}` everywhere else. I would keep it the same way here for consistency